src/components/ContentPreview/ContentPreview.js

```diff
@@ -48,6 +48,7 @@ type Props = {
     useHotkeys: boolean,
     showSidebar?: boolean,
     hasSidebar: boolean,
+    hasCustomBranding?: boolean,
```

Comment:
Could setting all this UI to dark be potentially useful for platform devs? In that case we may want to rename this

Reply:
Also this may be available via the Terms of Service endpoint. Could be a cool enhancement to detect automatically if the prop isn't passed in.